internal/job/checkout.go

```diff
@@ -217,6 +217,11 @@ func (e *Executor) CheckoutPhase(ctx context.Context) error {
 		}
 	}
 
+	err = e.sendCommitToBuildkite(ctx)
+	if err != nil {
+		e.shell.OptionalWarningf("commit-metadata", err.Error())
```

Comment:
`commit-metadata` doesn't really feel like it represents what this is - perhaps. Can we fit the word `resolve` or something in there? `resolve-git-commit` maybe?

Reply:
To create an actual suggestion - had a look at some other equivalent strings, and they describe the thing that happened (`-unavailable`, `-disabled` etc)
```suggestion
		e.shell.OptionalWarningf("git-commit-resolution-failed", err.Error())
```